src/services/nnas/routes/provider.ts

```diff
@@ -89,16 +91,28 @@ router.get('/service_token/@me', async (request: express.Request, response: expr
 		return;
 	}
 
-	const tokenOptions = {
-		system_type: server.device,
-		token_type: 0x4, // * Service token
+	const systemType = serverDeviceToSystemType[server.device];
+	if (!systemType) {

```

Comment:
Forgot this in my last review, but another style nit for the common eslint stuff. We typically have white space separating variable declarations and blocks of code

Reply:
Eslint hasn't been merged yet

I also personally disagree, these lines of code are directly related so I feel they should be grouped